### ISSUE TITLE
TypeError: vis.Timeline is not a constructor

### ISSUE THREAD

**Issue by shosaco** [daattali/timevis#33]
When running a simple example, I get the error in the javascript console "TypeError: vis.Timeline is not a constructor[Learn More]" and the output doesn't show up.

The example is a module in a bigger app. This happens only on my Linux machine, not on my windows.
```
library(timevis)
ui <- function(id){
  ns <- NS(id)
  fluidPage(h3("Timeline:"),
            timevisOutput(ns("timeline")))
}

server <- function(input, output, session) {
    output$timeline <- renderTimevis(
      timevis()
    )
}

shinyApp(ui, server)
```

R version 3.3.2 (2016-10-31)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

**Comment by daattali** [comment]
Do the examples at http://visjs.org/timeline_examples.html work for you?
Have you tried or do you have access to a non-centos linux?

---
Dean Attali
President & CEO
AttaliTech Ltd
http://AttaliTech.com <http://attalitech.com>

**Comment by shosaco** [comment]
The CentOS machine is my host for the shiny app, a remote machine used for my webserver, so I basically cannot "navigate" to visjs.org to try it. If there is anything that I can try to find the root cause of this problem, please let me know. And thanks for the great package!

**Comment by daattali** [comment]
Ah ok I'll try to see how I can debug that, might be difficult and I'm also
away right now (at useR conference and then traveling) so might be a while

---
Dean Attali
President & CEO
AttaliTech Ltd
http://AttaliTech.com <http://attalitech.com>

**Comment by daattali** [comment]
Actually, if you know how to enable X11 forwarding, you could test it out
on the centos machine (and if you don't know how, it's worth it to learn,
it will be an extremely useful skill forever)

---
Dean Attali
President & CEO
AttaliTech Ltd
http://AttaliTech.com <http://attalitech.com>

**Comment by shosaco** [comment]
I just fixed it, this issue was related to issue #11 and #12, I had visNetwork V1.0.1 running, whereas the bug was fixed there with V1.0.2. Upgraded now and timevis is working fine.

Thank you very much! If your travels lead you to Munich, let me know!

**Comment by daattali** [comment]
Ah good so I didn't do anything bad :) I'm definitely going to spend some
time in Germany this month, but probably Berlin, not Munich, but thank you
:)

---
Dean Attali
President & CEO
AttaliTech Ltd
http://AttaliTech.com <http://attalitech.com>
